src/conversion/upcastdispatcher.js

```diff
@@ -301,25 +304,64 @@ export default class UpcastDispatcher {
 	}
 
 	/**
-	 * Checks if {@link #_removeIfEmpty} contains empty elements and remove them.
-	 * We need to do it smart because there could be elements that are not empty because contains
-	 * other empty elements and after removing its children they become available to remove.
-	 * We need to continue iterating over split elements as long as any element will be removed.
+	 * Registers that `split` element is a split part of the `original` element.
+	 *
+	 * Data set by this method is then used by {@link #_getSplitParts} and {@link #_removeEmptyElements}
+	 *
+	 * @private
+	 * @param {module:engine/model/element~Element} original The original element.
+	 * @param {module:engine/model/element~Element} split The split part element.
+	 */
+	_registerSplitPair( original, split ) {
```

Comment:
```suggestion
	_registerSplitPair( original, copy ) {
```

Reply:
I am not a huge fan of `split` name but I think it is better than `copy`. This is not a copy of that element, they are both split parts. If not `split`, I'd go with `originalPart` + `newPart`.

Reply:
Or `originalPart` and `splitPart`.